app/views/content/life-as-a-teacher/teaching-as-a-career/early-career-teachers.md

```diff
@@ -3,6 +3,8 @@ title: "Early career teachers (ECTs)"
 heading: "Early career teachers (ECTs)"
 description: |-
   Find out about the support early career teachers get when they start teaching, including mentor support and training based on the early career framework (ECF).
+promo_content:
+  - "content/life-as-a-teacher/teaching-as-a-career/block-promos"
```

Comment:
Need to remove old promo 

![image](https://github.com/user-attachments/assets/85ce4912-0b6e-41fd-b7b3-6af790380b53)